plugins/available/killrails.plugin.bash

```diff
@@ -0,0 +1,15 @@
+cite about-plugin 
+about-plugin 'Function to kill a daemonized Rails server.'
+
+# Quick function to kill a daemonized Rails server
+function killrails() {
```

Comment:
Could you please add the `about` and `group` meta information for the function as well? Example [here](https://github.com/Bash-it/bash-it/blob/master/plugins/available/git.plugin.bash#L5)

Sorry for not catching that earlier.